### ISSUE TITLE
(XP) launcher does not display the news

### ISSUE THREAD

**Issue by FoxbitDreamtail** [olauncher/olauncher#76]
My test rig is a Asus Inspire One AO531h with 2Gb of RAM, it's running Windows XP service pack 3 with all updates even POSReady updates applied, it's running Java 8u152, the rest of the launcher works fine it's just the news section which does not load.
The best performance I got from this hardware was Minecraft 1.12.2 with OptiFine, it was very inconsistent with the mechanical hard drive, I hope to smooth it out when I get an SSD.

**Comment by bigfoot547** [comment]
Do you have the launcher logs for us? Perhaps your JVM trust store is out of date?

**Comment by notbronwyn** [comment]
I had this issue with 8u152 (but on Vista x64)
try using the latest commit from github actions

**Comment by FoxbitDreamtail** [comment]
> I had this issue with 8u152 (but on Vista x64) try using the latest commit from github actions, it uses sway by default

How does one do that?

**Comment by FoxbitDreamtail** [comment]
> Do you have the launcher logs for us? Perhaps your JVM trust store is out of date?

Where would I find the launcher logs, I can find the game logs just fine but where would I find the launcher's logs

**Comment by notbronwyn** [comment]
This is the exact same issue i had
Blame JfxWebkit

bigfoot is probably going to push a hotfix

**Comment by bigfoot547** [comment]
Yeah, it's the JavaFX bug. It was fixed in 98abde5fc64d93905481a6ea7ca6e2db36563145, which will be in the next hotfix.

**Comment by FoxbitDreamtail** [comment]
I will be waiting for the release of this hotfix and will close this issue when I can confirm that the issue has, indeed been resolved.
